modules/docextract/lib/refextract_kbs.py

```diff
@@ -194,7 +193,6 @@ def institute_num_pattern_to_regex(pattern):
     """
     simple_replacements = [
                             ('9',    r'\d'),
-                            ('9+',   r'\d+'),
                             ('w+',   r'\w+'),
```

Comment:
per comment above, I don't actually see a '+' in the kb, so is this line superfluous?